### ISSUE TITLE
update i18n tokens

### ISSUE THREAD

**Issue by pngwn** [gradio-app/gradio#1078]
In a,mongset the 3.0 work, we have added lots of new 'keywords' in various places. Often as default values (in case a kwarg isn't passed in). We need to go through all of these and swap any hardcoded values for i18n tokens, add them to the ebnlish 18n document and figure out how we can go about updating all of the other translations.

**Comment by abidlabs** [comment]
Yup sounds good. This is important, but I don't think this is a blocker for 3.0 so not adding to the 3.0 milestone. We can do this as part of revamped i18n efforts, which include #785 as well

**Comment by abidlabs** [comment]
@dawoodkhan82 I believe you worked on the i81n can you add more details here so that the community knows how to contribute translations for the new keywords?

**Comment by jannesblobel** [comment]
Hi together. 
First of all, sorry, @abidlabs,  that I opened the PR directly. I didn't see this Issue/ Conversation. 
We want to chip in and provide Gradio with a UI for contributors to add translations with [inlang/inlang](https://github.com/inlang/inlang). Please take a look as an example https://inlang.com/editor/github.com/inlang/example.
or my forked Gradio project https://inlang.com/editor/github.com/jannesblobel/gradio
